chiron/states.py

```diff
@@ -272,8 +306,16 @@ def get_reduced_potential(
         ) / unit.AVOGADRO_CONSTANT_NA
         # log.debug(f"reduced potential: {reduced_potential}")
         if self.pressure is not None:
-            reduced_potential += self.pressure * self.volume
+            self.volume = (
+                sampler_state.box_vectors[0][0]
+                * sampler_state.box_vectors[1][1]
+                * sampler_state.box_vectors[2][2]
+            ) * unit.nanometer**3
```

Comment:
_box_vectors have units attached, but the getter converts to the our chosen unit system and removes  them.  I think we need to refactor such that we can export variables with their units or without, at least for the public facing part.

Reply:
Issue #31 mentions this. It will be part of the refactor to openff-units and keeping units attached to quantities as much as possible (e.g., stripping them when passing to jitted functions, but reattaching to outputs).